README.md

```diff
@@ -112,8 +115,8 @@ The pipeline can be run using any of the following `make` commands:
 
 - Research Luxembourg COVID-19 Taskforce WP6, in the person of Alexander Skupin: supervision and coordination. Contact: alexander.skupin@uni.lu.
 - Daniele Proverbio: R_eff code development, website ideation and content creation
-- [Atte if we include his code]
-- [credits to R3 team for website development and deployment]
+- Atte Aalto: R_t code development
```

Comment:
maybe add the github handles as well. and I;d say to add the UL, the LCSB, and the Research Luxembourg logos